File: 250301/체질량지수/bmi.py
user = input()
arr = user.split()

h = int(arr[0])
w = int(arr[1])

b = int((10000 * w) / (h * h))

if 1 <= h <= 1000 and 1 <= w <= 1000 :
    if b >= 25 :
        print(b)
        print("Obesity")
    else :
        print(b)
else : 
    print("error")

File: 250721/체질량지수/bmi.swift
import Foundation

let input = readLine()!.split(separator: " ").map {Int(String($0))! }
let h = input[0]
let w = input[1]

if (1 <= h && h <= 1000) && (1 <= w && w <= 1000) {
    let b = ((10000 * w) / (h * h))
    print(b)
    if b >= 25 {
        print("Obesity")
    }
}